samples/subsys/ipc/ipc_service/static_vrings/boards/imx943_evk_mimx94398_m7_1.overlay

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2025 NXP
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/dt-bindings/ipc_service/static_vrings.h>
+
+/ {
+	ipc {
+		ipc0: ipc0 {
+			compatible = "zephyr,ipc-openamp-static-vrings";
+			memory-region = <&dram_m33s_m71_ipc0>;
+			mboxes = <&mu_m33s_m71_for_m71 0>, <&mu_m33s_m71_for_m71 1>;
+			mbox-names = "tx", "rx";
+			role = "host";
+			status = "okay";
+		};
+
+		ipc1: ipc1 {
+			compatible = "zephyr,ipc-openamp-static-vrings";
+			memory-region = <&dram_m33s_m71_ipc1>;
+			mboxes = <&mu_m33s_m71_for_m71 2>, <&mu_m33s_m71_for_m71 3>;
+			mbox-names = "tx", "rx";
+			role = "host";
+			zephyr,priority = <1 PRIO_COOP>;
+			zephyr,buffer-size = <128>;
+			status = "okay";
+		};
+
+		//ipc0: ipc0 {
+		//	compatible = "zephyr,ipc-openamp-static-vrings";
+		//	memory-region = <&dram_m70_m71_ipc0>;
+		//	mboxes = <&mu_m70_m71_for_m71 0>, <&mu_m70_m71_for_m71 1>;
+		//	mbox-names = "tx", "rx";
+		//	role = "host";
+		//	status = "okay";
+		//};
```

Comment:
Pleae do not add commented code. If the code is not used should be deleted. If the code serves as a documentation should be placed in documentation section.

Reply:
> Pleae do not add commented code. If the code is not used should be deleted. If the code serves as a documentation should be placed in documentation section.

okay, got it.